docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -20,272 +20,136 @@ Current binder implementations include:
-
 ==== Basic Usage
 
-==== Samples
-
-==== Error Channels
+. Method 1: Connection string based usage
 
-*_consumer error channel_*
+**  Create [Azure Event Hubs][create-event-hubs].
+After creating the Azure Event Hub, you can create your own Consumer Group or use the default "$Default" Consumer Group.
 
-this channel is open by default, you can handle the error message in this way:
-
-----
-    // Replace destination with spring.cloud.stream.bindings.input.destination
-    // Replace group with spring.cloud.stream.bindings.input.group
-    @ServiceActivator(inputChannel == "{destination}.{group}.errors")
-    public void consumerError(Message<?> message) {
-        LOGGER.error("Handling customer ERROR: " + message);
-    }
-----
-
-*_producer error channel_*
-
-this channel is not open by default, if you want to open it. You need to add a configuration in your application.properties, like this:
-
-----
-spring.cloud.stream.default.producer.errorChannelEnabled=true
-----
-
-you can handle the error message in this way:
-
-----
-    // Replace destination with spring.cloud.stream.bindings.output.destination
-    @ServiceActivator(inputChannel == "{destination}.errors")
-    public void producerError(Message<?> message) {
-        LOGGER.error("Handling Producer ERROR: " + message);
-    }
-----
-
-===== Batch Consumer Sample
-
-====== Configuration Options
-
-To enable the batch consumer mode, you should add below configuration
+**  Create [Azure Storage][create-azure-storage] for checkpoint usage.
 
+**  Update [application.yaml][application.yaml].
 [source,yaml]
 ----
-spring:
-  cloud:
-    stream:
-      bindings:
-        consume-in-0:
-          destination: {event-hub-name}
-          group: [consumer-group-name]
-          consumer:
-            batch-mode: true 
-      eventhubs:
-        bindings:
-          consume-in-0:
-            consumer:
-              checkpoint:
-                mode: BATCH # or MANUAL as needed
-              batch:
-                max-size: 2 # The default value is 10
-                max-wait-time: 1m # Optional, the default value is null
+    spring:
+      cloud:
+        azure:
+          eventhubs:
+            # Fill event hub namespace connection string copied from portal
+            connection-string: [eventhub-namespace-connection-string]
+            # Fill checkpoint storage account name, access key and container
+            processor:
+              checkpoint-store:
+                container-name: [checkpoint-container]
+                account-name: [checkpoint-storage-account]
+                account-key: [checkpoint-access-key]
+        stream:
+          function:
+            definition: consume;supply
+          bindings:
+            consume-in-0:
+              destination: [eventhub-name]
+              group: [consumer-group]
+            supply-out-0:
+              destination: [the-same-eventhub-name-as-above]
 ----
 
-======= Consume messages in batches
```

Comment:
For the feature parts, you should move it the "sample" section and give a formal introduction about how to use those features.